skyportal/handlers/photometry.py

```diff
@@ -68,6 +71,23 @@ def post(self):
             DBSession().add(p)
             DBSession().flush()
             ids.append(p.id)
+        if 'thumbnails' in data:
+            for thumb in data['thumbnails']:
+                file_uri = os.path.abspath(
+                    f'static/thumbnails/{source.id}_{thumb["type"]}.png')
```

Comment:
Again, `type` is provided by the user, so I think we should just generate our own filename here.  If you want to use type in there, I would first verify the type.

Reply:
This is no longer used